libvast/test/option_declaration_set.cpp

```diff
@@ -83,8 +83,8 @@ TEST(data type parsing) {
   check_fail_option(*x, "X");
   MESSAGE("Test string");
   x = decl.find("string");
-  check_option(*x, "\"2\"", "2");
```

Comment:
Removing the quotations defeats the point of the test. Is this no longer supported by the current implementation?

Reply:
Ok, you are right. When we do not use the command line we need the VAST parser for unescaping.
I will add your code snippet.